internal/lang/functions_test.go

```diff
@@ -1231,7 +1242,7 @@ func TestFunctions(t *testing.T) {
 	}
 
 	experimentalFuncs := map[string]experiments.Experiment{}
-	experimentalFuncs["defaults"] = experiments.ModuleVariableOptionalAttrs
```

Comment:
I didn't see the removal of this function from the experiments mentioned anywhere in the description. Just wanted to check it was deliberate.

Reply:
This `defaults` function was associated with a long-concluded experiment and hasn't existed since that experiment concluded, so this entry wasn't actually doing anything -- there is no function named `defaults` in the function table now anyway.

Normally we catch and remove these as part of stabilizing an experiment because the test below would find that the function is callable even without the experiment enabled, but this one got missed because the function was deleted rather than stabilized (the built-in support for optional attribute defaults replaced it) and so the testing loop didn't actually visit this function name at all.

(Maybe we should have an extra rule here to fail the test if there's an experimental function mentioned that doesn't actually exist, but I'd rather keep that for a separate PR.)